src/dex-ui-components/base/FormInputList/FormInputList.tsx

```diff
@@ -36,26 +49,32 @@ export function FormInputList<FormType extends FieldValues, FormPath extends Arr
     <>
       {fields.map((field, index) => {
         return (
-          <Flex key={index} alignItems="end">
+          <Flex key={index} alignItems="end" gap="0.5rem" paddingRight="0.5rem">
             <FormInput
               flex="5"
               inputProps={{
                 id: field.id,
                 type: "text",
-                label: "Member wallet address",
-                placeholder: "Enter URL",
+                label: fieldLabel,
+                placeholder: fieldPlaceholder,
                 register: { ...register(`${formPath}.${index}.value` as Path<FormType>) },
               }}
             />
-            <Button marginLeft="0.25rem" marginBottom="0.5rem" onClick={() => handleRemoveInputClicked(index)}>
-              Remove
-            </Button>
+            <IconButton
+              size="xs"
+              variant="link"
+              marginLeft="0.25rem"
+              marginBottom="1.2rem"
+              aria-label="remove-multisig-member"
+              onClick={() => handleRemoveInputClicked(index)}
+              icon={<CancelledStepIcon fill={Color.Grey_Blue._500} options={{ height: 21, width: 21 }} />}
```

Comment:
I recommend updating the `CancelledStepIcon` to use the `createIcon` function from `@chakra-ui/react`. This allows the icon to inherit all of the ChakraUI styling options. You will be able to directly style the components without using intermediary props, e.g. `<CancelledStepIcon boxSize="4">`.

Reply:
Done